test/data/repositories/fedora-34.json

```diff
@@ -0,0 +1,26 @@
+{
+  "x86_64": [
+    {
+      "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/public/f34/f34-x86_64-devel-20210414/",
```

Comment:
@ondrejbudai could you please add `name: xxxx` here and we can have source name in osbuild-composer source list. Thanks.

Reply:
And we need copy this file into `/etc/osbuild-composer/repositories/` in `provision.sh`

Reply:
I force-pushed a fix, repositories now have names. The file should be already being copied, see: https://github.com/osbuild/osbuild-composer/blob/27c5aafeca102fb0634866c4adb2bb938d3262e7/tools/provision.sh#L52 I also verified this on Fedora 34 Schutzbot test machine:

```
$ ls /etc/osbuild-composer/repositories/
centos-8.json  centos-stream-8.json  fedora-32.json  fedora-33.json  fedora-34.json
```